README.md

```diff
@@ -4,165 +4,147 @@ SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org
+     $ cd gnbsim
+     $ make docker-build #this will create docker image
+
+    If you want to run gNBSim along with other SD-Core Network Functions then use aether onRamp to deploy all network functions.
+    If you want to run gNBSim as a standalone tool then deploy gNBSim using onRamp. 
     Enter gnbsim pod using kubectl exec command and run following commands, 
-    
-    $ ./gnbsim
-    
-    Note: By default, the gNB Sim reads the configuration from 
-    /gnbsim/config/gnb.conf file. To provide a different configuration file,
-    use the below command
 
-    $ ./gnbsim --cfg config/gnbsim.yaml
+    All these steps are explained in detail on [AIAB documentation](https://docs.sd-core.opennetworking.org/master/developer/aiab.html)
+
+# Supported features
+
+   Supported 3gpp procedures
+
+    - UE Registration
+    - UE Initiated PDU Session Establishment
+    - UE Initiated De-registration
+    - AN Release
+    - UE Initiated Service Request
+    - N/W triggered PDU Session Release
+    - UE Requested PDU Session Release
+    - N/W triggered UE Deregistration
 
-All these steps are explained in detail on [AIAB documentation](https://docs.sd-core.opennetworking.org/master/developer/aiab.html)
 
-## Step 4: Optionally launching profiles through HTTP APIs
+   Supported System level features
 
-    gNBSim can process HTTP Requests to launch profiles. For example running the
-    below curl command will launch a profile in gNBSim
-   
-    $ curl -i -X POST 127.0.0.1:6000/gnbsim/v1/executeProfile -H 'Content-Type: application/json' -d '{"profileType":"nwreqpdusessrelease","profileName":"profile8","enable":true,"gnbName":"gnb1","startImsi":"208930100007497","ueCount":1,"opc":"981d464c7c52eb6e5036234984ad0bcf","key":"5122250214c33e723a5dd523fc145fc0","sequenceNumber":"16f3b3f70fc2","defaultAs":"192.168.250.1","plmnId":{"mcc":"208","mnc":"93"}}'
+    - Gnbsim can generate and send user data packets (ICMP echo request)
+      and process downlink user data (ICMP echo response) over the established data
+      plane path (N3 Tunnel).
+    - Executing all enabled profiles in parallel or in sequential order.
+    - Timeout for each call flow within profile
+    - Logging summary result
+    - HTTP API to execute profile
+    - Configure number of data packets to be sent and time interval between consecutive packets
+    - Configure AS (Application Server) address. This is used to send data packets
+    - Run gNBSim with single Interface or multi interface
+    - Support of Custom Profiles
+    - Delay between Procedures
+    - Timeout for every profile
+    - Logic to calculate latency per transaction/ operation
+    - Support retransmission of Service Request Message
 
-# Pending Feature List
+# Pending Features
+
+   Data Testing Features
+
+    - Provision data interface to gNBSim Container/POD/executable for data traffic testing
+    - Triggering downlink data from gNB Sim (CI/CD feature as well)
+
+    3gpp features for gNodeB Simulator
```

Comment:
I think this should not have indentation (Take a look at the rendered markdown output)
```suggestion
   3gpp features for gNodeB Simulator
```